esp32c3-hal/Cargo.toml

```diff
@@ -42,11 +42,14 @@ embedded-graphics = "0.7.1"
 esp-backtrace     = { version = "0.6.0", features = ["esp32c3", "panic-handler", "exception-handler", "print-uart"] }
 esp-hal-smartled  = { version = "0.1.0", features = ["esp32c3"], path = "../esp-hal-smartled" }
 esp-println       = { version = "0.4.0", features = ["esp32c3"] }
+hmac              = { version = "0.12", default-features = false}
+sha1              = { version = "0.10", default-features = false}
 sha2              = { version = "0.10.6", default-features = false}
 smart-leds        = "0.3.0"
 ssd1306           = "0.7.1"
 static_cell       = "1.0.0"
 crypto-bigint = {version = "0.5.0-pre.3",default-features = false}
+hex-literal       = "0.4"
```

Comment:
Don't think this is needed, either.

Reply:
I was using them before settling stress test so I will remove it.